client/draft_README.md

```diff
@@ -54,15 +52,14 @@ HairrKeepers helps its customers find the right hair experts in any city within
 </details>
 
 <details>
- <summary> 🎨 Wireframes</summary>
+ <summary> 🎨 Figma Mockup Designs by Cynthia Eddy</summary>
 
-  <!-- TODO: [ ] - Requesting Section Completion By Cynthia -->
+ <!-- TODO: [ ] - Update to include the Mobile Figma Mockup Designs By Cynthia -->
```

Comment:
Hi @cynthiaeddy!  Updated to reflect the latest action for your attention, per our discussion at our last meeting.  Thank you again!